packages/location-field/src/styled.js

```diff
@@ -22,7 +22,8 @@ export const DropdownButton = styled(BaseButton)`
 `;
 
 export const DropdownContainer = styled.span`
-  position: relative;
+  position: contents;
```

Comment:
One of these CSS changes causes the dropdown suggestion list to be limited by the width of the text field, and I don't think we want that. On desktop views, the dropdown width should extend to accommodate as much of its contents as possible.
![image](https://user-images.githubusercontent.com/56846598/135472560-5afceefe-ed48-48b3-a745-6e8a28ee9c9d.png)

Reply:
I had to make some changes in https://github.com/opentripplanner/otp-react-redux/pull/462 to accommodate this but thank you for suggesting in the end it is much nicer to be able to display the entire title! Updated in be62bb29b9db033a82706971d3c4031a337aef23